financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsConsumerSessionRepository.kt

```diff
@@ -113,6 +119,18 @@ private class FinancialConnectionsConsumerSessionRepositoryImpl(
         }
     }
 
+    override suspend fun attachLinkConsumerToLinkAccountSession(
+        consumerSessionClientSecret: String,
+        clientSecret: String,
+    ): AttachConsumerToLinkAccountSession {
+        return consumersApiService.attachLinkConsumerToLinkAccountSession(
+            consumerSessionClientSecret = consumerSessionClientSecret,
+            clientSecret = clientSecret,
+            requestSurface = CONSUMER_SURFACE,
+            requestOptions = provideApiRequestOptions(useConsumerPublishableKey = false),
```

Comment:
we don't want to use the consumer key here right?

Reply:
Yep, we only use the consumer publishable key in the following synchronize call and thereafter.